src/nrtk_explorer/app/transforms.py

```diff
@@ -75,21 +76,49 @@ def __init__(self, server):
         self.state.transforms = [k for k in self._transforms.keys()]
         self.state.current_transform = self.state.transforms[0]
 
-        # Transform enabled control ##
+        # Transform enabled control ###
+        def just_enabled_transform(old, new):
+            return old is False and new
+
+        def turn_on_transform_columns(_, __):
+            if "transformed" not in self.state.visible_columns:
+                self.state.visible_columns = self.state.visible_columns + ["transformed"]
```

Comment:
`self.state.visible_columns += ["transformed"]`

Reply:
When I do that the UI does not update (because `+=` mutates the list in place I think)

Reply:
I see, then what about `self.state.visible_columns = [*self.state.visible_columns, "transformed"]`

Reply:
Motion adopted!